src/main/webapp/user_profile_script.js

```diff
@@ -52,10 +104,105 @@ function closeModal() {
     modal.style.display = "none";
 }
 
+function closeEditModal() {
+    var modal = document.getElementById("editTaskModal");
+    modal.style.display = "none";
+}
+
 // If the user clicks outside of the modal, closes the modal directly
 window.onclick = function(event) {
     var modal = document.getElementById("createTaskModal");
     if (event.target == modal) {
         modal.style.display = "none";
     }
+    var editModal = document.getElementById("editTaskModal");
+    if (event.target == editModal) {
+        editModal.style.display = "none";
+    }
+}
+
+async function displayNeedHelpTasks() {
+    const queryURL = "mytasks?keyword=Owner";
```

Comment:
I'd recommend using an absolute URL, almost always: `/mytasks?...`

Reply:
I forgot to add the slash when I change the url that handles the POST request. I will add it in the new commit.